app/assets/templates/status-message-location_tpl.jst.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if location}}
   <div class='near-from'>
-    {{ t "publisher.near_from" location=location}}
+    <a href="/posts/{{id}}">{{ t "publisher.near_from" location=location}}</a>
```

Comment:
@jhass I'm not sure if I get that right. Do you want us to style the location in the stream with `cursor: pointer` and `text-decoration: underline` that should get visible via hovering? What about the link to the single post view (that's where the map is right now)? Should it stay?